lib/statsd.rb

```diff
@@ -140,7 +201,7 @@ def send(stat, delta, type, sample_rate=1)
       stat.gsub!(/::/, ".".freeze)
       stat.gsub!(RESERVED_CHARS_REGEX, "_".freeze)
 
-      msg = ""
+      msg = String.new
```

Comment:
What's the reason for this change? I didn't see anything in the commit messages. (My guess it's to be compatible with frozen-by-default string literals in a future Ruby version.)